--- android_icu4j/src/main/java/android/icu/text/BidiClassifier.java	2018-09-07 16:23:24.888261363 +0100
+++ android_icu4j/src/main/java/android/icu/text/BidiClassifier.java	2018-09-07 16:23:49.056381325 +0100
@@ -83,10 +83,9 @@
      *
      * @param c Code point to be classified.
      * @return An integer representing directional property / Bidi class for the
-     *         given code point <code>c</code>, or Bidi.CLASS_DEFAULT=UCharacter.getIntPropertyMaxValue(UProperty.BIDI_CLASS)+1
+     *         given code point <code>c</code>, or UCharacter.getIntPropertyMaxValue(UProperty.BIDI_CLASS)+1
      *         to signify that there is no need to override the standard Bidi class for
      *         the given code point.
-     * @see Bidi#CLASS_DEFAULT
      */
     public int classify(int c) {
         return Bidi.CLASS_DEFAULT;
