--- android_icu4j/src/main/java/android/icu/text/CaseMap.java	2018-09-03 14:54:02.236325503 +0100
+++ android_icu4j/src/main/java/android/icu/text/CaseMap.java	2018-09-03 14:53:51.400265158 +0100
@@ -239,7 +239,6 @@
          *
          * @return an options object with this option.
          * @see UCharacter#TITLECASE_NO_LOWERCASE
-         * @see #adjustToCased()
          */
         public Title noLowercase() {
             return new Title(internalOptions | UCharacter.TITLECASE_NO_LOWERCASE);
@@ -250,9 +249,7 @@
          * does not adjust the titlecasing BreakIterator indexes;
          * titlecases exactly the characters at breaks from the iterator.
          *
-         * <p>By default, titlecasing will take each break iterator index,
-         * adjust it to the next relevant character (see {@link #adjustToCased()}),
-         * and titlecase that one.
+         * <p>By default, titlecasing will take each break iterator index.
          *
          * <p>Other characters are lowercased.
          *
