--- android_icu4j/src/main/java/android/icu/text/CaseMap.java	2018-10-22 05:12:26.772613338 +0100
+++ android_icu4j/src/main/java/android/icu/text/CaseMap.java	2018-10-22 05:12:45.412674957 +0100
@@ -235,7 +235,6 @@
          *
          * @return an options object with this option.
          * @see UCharacter#TITLECASE_NO_LOWERCASE
-         * @see #adjustToCased()
          */
         public Title noLowercase() {
             return new Title(internalOptions | UCharacter.TITLECASE_NO_LOWERCASE);
@@ -246,9 +245,7 @@
          * does not adjust the titlecasing BreakIterator indexes;
          * titlecases exactly the characters at breaks from the iterator.
          *
-         * <p>By default, titlecasing will take each break iterator index,
-         * adjust it to the next relevant character (see {@link #adjustToCased()}),
-         * and titlecase that one.
+         * <p>By default, titlecasing will take each break iterator index.
          *
          * <p>Other characters are lowercased.
          *
