--- android_icu4j/src/main/java/android/icu/text/MeasureFormat.java	2021-01-05 16:28:14.205297829 +0000
+++ android_icu4j/src/main/java/android/icu/text/MeasureFormat.java	2021-01-05 16:28:33.829426812 +0000
@@ -61,12 +61,6 @@
  * A formatter for Measure objects.
  *
  * <p>
- * <strong>IMPORTANT:</strong> New users are strongly encouraged to see if
- * {@link NumberFormatter} fits their use case.  Although not deprecated, this
- * class, MeasureFormat, is provided for backwards compatibility only.
- * <hr>
- *
- * <p>
  * To format a Measure object, first create a formatter object using a MeasureFormat factory method. Then
  * use that object's format or formatMeasures methods.
  *
